### PR TITLE
Add new side panel control components

### PR DESCRIPTION
**Pull request by tienhoah**
- **Remove hamburger menu icon** from header for panel show/hide state
- Replace with new **side control component**:
   - Separate **Show/hide button**.
   - **Clear off selected site** and back to summary info.
Adjust some UX to fit with new side control.

### CODE REVIEW

**Comment by tienhoah** [comment]
refers to #120 for mockup image

File: src/components/DashBoard.js [hunk 1]
```diff
@@ -237,7 +237,7 @@ const DashBoard = ({
           </Box>
         </Paper>
       </Grid>
-      {dashboardControl}
+      {sidePanelCOntrol}
```
Comment (brianbancroft) [inline src/components/DashBoard.js]: 🤖

File: src/components/DrawerDashBoard.js [hunk 2]
```diff
@@ -81,14 +94,14 @@ const DrawerDashBoard = ({
 }) => {
   const classes = drawerStyleProperties();
 
-  const collapseDashboardMenu = (
+  const collapseSidePanel = (
     <Box className={classes.collapseButtonProperty}>
-      <ThemeProvider theme={theme.dashboardControl}>
+      <ThemeProvider theme={theme.sidePanelCOntrol}>
```
Comment (brianbancroft) [inline src/components/DrawerDashBoard.js]: 🤖

File: src/components/DashBoard.js [hunk 3]
```diff
@@ -198,7 +198,7 @@ const DashBoard = ({
   const fullMapControl = <Box className={classes.zoomOutIconWrapperProperty}>{fullMapToggle}</Box>;
   const zoomToControl = <Box className={classes.zoomToIconWrapperProperty}>{zoomToSelectSite}</Box>;
 
-  const dashboardControl = (
+  const sidePanelCOntrol = (
```
Comment (brianbancroft) [inline src/components/DashBoard.js]: 🤖

**Pull request merged**